app/helpers/quadicon_helper.rb

```diff
@@ -32,6 +32,7 @@ module QuadiconHelper
     'terminated'                => {:fonticon => 'pficon pficon-off', :background => '#CC0000'},
     'off'                       => {:fonticon => 'pficon pficon-off', :background => '#CC0000'},
     'template'                  => {:fonticon => 'pficon pficon-template', :background => '#336699'},
+    'powering_up'               => {:fileicon => 'svg/currentstate-powering-up.svg', :background => '#F0AB00'},
```

Comment:
The background image doesn't really make sense with the SVG, only with a fonticon. My suggestion would be `fa fa-play` but maybe @epwinchell can tell you a better one.

Reply:
@skateman We are using this SVG in vm details view. Do we want to be consistent?

Reply:
@pkliczewski consistent with what? @martinpovolny is trying to pull these fonticon+color+background definitions into textual summaries :wink: also the triangle seems as an already deprecated patternfly icon, let's wait for Eric's opinion.

Reply:
@skateman Consistent with vm detail view where this svg is used already.

Reply:
The SVG with the triangle is AFAIK deprecated and we went to the `pficon-off` + background color layout from the play/pause/stop/etc icons, at least on quadicons. The summary screens (VM detail) are still using the old icons, but that's just a question of time to change them too.

Reply:
I'd suggest using pficon-on with #FF9900 as the background (orange)

Reply:
'powering_up'               => {:fonticon => 'pficon pficon-on', :background => '#FF9900'},